decompressed/gui_file/www/gateway-snippets/footer.lp

```diff
@@ -30,11 +32,13 @@ local html = {}
 html[#html+1] =		'<div id="footer" class="row">'
 html[#html+1] =			'<div class="copyright">'
 html[#html+1] =				format('<p>&copy; Technicolor %s </p>', current_year)
-html[#html+1] =				format('<p>GUI Version [ %s ] 2018 ',cui.gui_version)
+html[#html+1] =				format('<p>%s %s GUI Version', cui.provisioning_code, cui.variant_friendly_name)
 							if session:hasAccess("/modals/changelog.lp") then
-html[#html+1] =					'<a href data-toggle="modal" data-remote="modals/changelog.lp">Changelog</a>'
+html[#html+1] =					format(' <a href data-toggle="modal" data-remote="modals/changelog.lp">%s</a>', cui.gui_version)
+							else
+html[#html+1] =					format(' %s', cui.gui_version)
 							end
-html[#html+1] =				'</p>'
+html[#html+1] =				format(' %s </p>', "2018")
```

Comment:
why %s for an hardcoded 2018 string? :D

Reply:
Figured this is going to be a variable in future.